packages/docs/core-concepts/index.md

```diff
@@ -14,6 +14,21 @@ export const useStore = defineStore('main', {
 
 This _name_, also referred as _id_, is necessary and is used by Pinia to connect the store to the devtools. Naming the returned function _use..._ is a convention across composables to make its usage idiomatic.
 
+Note that you can also define a store using a setup function instead, similar to Vue's Composition API:
+
+```js
+import { defineStore } from 'pinia'
+
+// useStore could be anything like useUser, useCart
+// the first argument is a unique id of the store across your application
+export const useStore = defineStore('main', () => {
+  return {
```

Comment:
I can't get it working with the return keyword in that spot... always used it like in https://github.com/antfu/vitesse/blob/main/src/stores/user.ts

Reply:
It works.
You probably didn't wrap your state with "toRefs" so you lost the reactivity.